Cargo.toml

```diff
@@ -57,6 +57,7 @@ assert_fs = { version = "1.1.0" }
 argfile = { version = "0.2.0" }
 bincode = { version = "2.0.0" }
 bitflags = { version = "2.5.0" }
+bitvec = { version = "1.0.1" }
```

Comment:
Can we disable the default features and only enable std (I don't think we need the atomic support). This, unfortunately, doesn't help with the new dependencies